lib/components/user/form-navigation-buttons.tsx

```diff
@@ -23,7 +23,7 @@ export interface ButtonType {
   /** Triggered when the button is clicked. */
   onClick?: MouseEventHandler<Button>
   /** The text to display on the button (JSX elements accepted). */
-  text?: ReactElement
+  textOverride?: ReactElement
```

Comment:
Not sure why you'd want to rename this field, as there is no default text that is set if this attribute is not defined.

Reply:
I thought @AdrianaCeric said that there was a default?

Reply:
I think naming can go either way. what matters is functionality. We can rename and refactor some things in FormNavigationButtons but for now its circling around the point of this PR